src/components/v2/organisms/BasicInputFormLocal.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useRef, useState } from 'react'
+import { useEffectOnce } from 'react-use'
+import Form from '../../../shared/components/molecules/Form'
+import { ButtonProps } from '../../../shared/components/atoms/Button'
+import { FormRowProps } from '../../../shared/components/molecules/Form/templates/FormRow'
+
+interface EmojiInputFormProps {
+  prevRows?: FormRowProps[]
+  defaultInputValue?: string
+  placeholder?: string
+  defaultEmoji?: string
+  defaultIcon: string
+  inputIsDisabled?: boolean
+  submitButtonProps: ButtonProps & {
+    label: React.ReactNode
+    spinning?: boolean
+  }
+  onSubmit: (input: string, emoji?: string) => void
+}
+
+const BasicInputFormLocal = ({
+  defaultInputValue = '',
+  prevRows = [],
+  placeholder,
+  submitButtonProps,
+  inputIsDisabled,
+  onSubmit,
+}: EmojiInputFormProps) => {
+  const inputRef = useRef<HTMLInputElement>(null)
+  const [value, setValue] = useState(defaultInputValue)
+
+  // seems to sometimes do some update on un-mounted component (called in openModal which closes itself)
+  useEffectOnce(() => {
```

Comment:
Focusing or not should be determined by its parent component. So please use ref so the parent can access the element and focus